### PR TITLE
New PDF Importer for Bank11

### PR DESCRIPTION
**Pull request by ZfT2**

### CODE REVIEW

File: name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/Bank11PDFExtractor.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+            String date = matcherMap.get("date");
+
+            date += context.get(CONTEXT_KEY_YEAR);
+
+            transaction.setDateTime(asDate(date));
+            transaction.setAmount(asAmount(matcherMap.get("amount")));
+            transaction.setCurrencyCode(asCurrencyCode(context.get(CONTEXT_KEY_CURRENCY)));
+        };
+    }
```
Comment (Nirus2000) [inline name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/Bank11PDFExtractor.java:106]: Hello @ZfT2 
1.)
I think you keep making the same mistake...

2.)
You dont need the BiConsumer.... again....

3.)
You dont need `Map<String, String> context = type.getCurrentContext();`

add follow like -> [here](https://github.com/portfolio-performance/portfolio/blob/master/name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/DkbPDFExtractor.java#L723:L738) <-

`.documentContext("currency", "year") //`

then you can simply use `v.get("currency")` or `v.get("year")` to retrieve the string, as it is already parsed in the document.

**Review by Nirus2000 — CHANGES_REQUESTED** [review]

**Comment by ZfT2** [comment]
Sorry... hope it is now resolved correctly.

**Comment by Nirus2000** [comment]
Perfect 🥇

**Comment by buchen** [comment]
Whenever @Nirus2000 says 'perfect', I just merge it. Thanks @ZfT2

**Pull request merged**